app/models/forms/bulk_upload_sales/prepare_your_file.rb

```diff
@@ -27,6 +27,10 @@ def template_path
         "/files/bulk-upload-sales-template-v1.xlsx"
       end
 
+      def specification_path
+        "/files/Sales 2022-23 Bulk Upload Specification.csv"
```

Comment:
these files don't actually appear to be CSV files, they're showing up as binary for this review and not text. therefore my guess is that are actually excel spreadsheets and should therefore honour the extension rather than masquerading the file as a CSV when it is not

Reply:
Yep, they're direct from https://core.communities.gov.uk/content/GuidesAndManuals.html - you're right, they're not real csvs... was initially hesitant to change these files to be different to what they are on that page though, and afaik I don't have access to change that one. I've just seen the templates are different in our service to what they are on that site however so I'll go ahead and make that change